pkg/controller.v1beta1/trial/trial_controller_status.go

```diff
@@ -23,20 +23,14 @@ import (
 )
 
 const (
-	// For Trials
-	TrialCreatedReason            = "TrialCreated"
-	TrialRunningReason            = "TrialRunning"
-	TrialSucceededReason          = "TrialSucceeded"
-	TrialMetricsUnavailableReason = "MetricsUnavailable"
-	TrialFailedReason             = "TrialFailed"
-
```

Comment:
I think we want to keep these constants in `trial_controller_status`, similar to Suggestion Controller: https://github.com/kubeflow/katib/blob/master/pkg/controller.v1beta1/suggestion/suggestion_controller_status.go#L26